public/computer-lab/lab.css

```diff
@@ -0,0 +1,8 @@
+document, body {
```

Comment:
in CSS [type selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Type_selectors) like "body" or "div" apply the rules to any/all elements which match that type, which means the "type" should always be a valid HTML element, there is no "document" element. so this should just be `body {`

in JavaScript `document` is the global object which exposes most of the functionality behind the browsers [DOM API](https://developer.mozilla.org/en-US/docs/Web/API/HTML_DOM_API)